source/manual/common-aws-tasks-for-2nd-line-support.html.md

```diff
@@ -247,18 +121,18 @@ decisions have been made.
 
 ## How are instances provisioned?
 
-We typically use Terraform config in [govuk-aws][] to provision infrastructure,
+We typically use Terraform config in [govuk-aws] to provision infrastructure,
 such as EC2 instances.
 
 There are a few exceptions to this, such as ad-hoc instances started from
 Concourse via the AWS CLI - these are mainly for data science projects.
 
-We use [userdata scripts][] to run commands on our instances at launch. These
+We use [userdata scripts] to run commands on our instances at launch. These
 scripts install various core bits of software needed by a particular instance
-and then typically use [govuk-puppet][] to provision our instances.
+and then typically use [govuk-puppet] to provision our instances.
 
 Finally, new instances send Jenkins their Fully Qualified Domain Name (FQDN)
-and puppet class. Jenkins automatically [deploys apps][] to newly provisioned
+and puppet class. Jenkins automatically [deploys apps] to newly provisioned
```

Comment:
I think "Jenkins" should be "GitHub Actions" here.

Reply:
Ah good point — so this is still true but only because the whole section is talking about the old EC2 stuff, which we should definitely call out otherwise it'll be confusing to anyone new. I've amended the heading and stuck a big note at the top of the section to hopefully address that. Thanks!